android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/RegisterFragment.kt

```diff
@@ -53,12 +56,16 @@ class RegisterFragment : Fragment() {
     container: ViewGroup?,
     savedInstanceState: Bundle?
   ): View {
+    appMainViewModel.retrieveRemoteIconsBitmap()
     return ComposeView(requireContext()).apply {
       setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
       setContent {
         val scope = rememberCoroutineScope()
         val scaffoldState = rememberScaffoldState()
-        val uiState: AppMainUiState = appMainViewModel.appMainUiState.value
+//        val uiState: AppMainUiState = appMainViewModel.appMainUiState.value
+        val uiState: AppMainUiState by remember {
+          mutableStateOf(appMainViewModel.appMainUiState.value)
+        }
```

Comment:
Do we need to `remember` a `MutableState`?

Reply:
I think we need to observe the state any time it is updated.

Reply:
@kitoto
That's ok as well.

Do you have a good sample for that?
Using live data or state?

Reply:
Since a`appMainUiState` we do not need to  create a new instance of a `MutabeState` to wrap it. Remove the `remember` lambda wrapper

Reply:
Okay, done